### ISSUE TITLE
Network initialization error in Testing

### ISSUE THREAD

**Issue by AhmedHisham99** [stanleyshly/EverybodyDanceNow#1]
![](https://user-images.githubusercontent.com/49320199/106517331-076dfd00-64e1-11eb-96cb-1a942869a47f.png)

**Comment by AhmedHisham99** [comment]
where can i download the "latest_net_G.pth" ??

i use test in sample_data

**Comment by stanleyshly** [comment]
You must train a model by yourself; I did not include pretrained models.

**Comment by AhmedHisham99** [comment]
where is the training data ?
is that in sample data ? but there are 3 images only 
or should run 'graph
-triain.py" ? but i can not find the data set used in that cell ?

**Comment by AhmedHisham99** [comment]
i open dance_subjects i see 5 .tar files should i train to all of them or what ?

**Comment by stanleyshly** [comment]
Train on only one, the openpose does not encode or and facial recognition information meaning it only work with one person in one outfit, just make sure to unzip to a train directory.

**Comment by AhmedHisham99** [comment]
I have same problem ! (when run train in the subject1 after unzip it)
I Found Checkpoints folder empty so that is the problem , now i don't know where to fill it .
![Screenshot from 2021-02-03 01-13-39](https://user-images.githubusercontent.com/49320199/106675134-3b1e5500-65bd-11eb-8f2b-5e4ce5c20b63.png)

![Screenshot from 2021-02-03 01-13-50](https://user-images.githubusercontent.com/49320199/106675146-41143600-65bd-11eb-901e-5294926ff254.png)

**Comment by stanleyshly** [comment]
Delete the continue train argument, it should fix the "not exists yet" error.

**Comment by AhmedHisham99** [comment]
after removing it this error came

![Screenshot from 2021-02-03 01-27-14](https://user-images.githubusercontent.com/49320199/106676275-36f33700-65bf-11eb-9bf7-1f94ab05a22c.png)

**Comment by AhmedHisham99** [comment]
I have confusion  about testing i see in sample data (test set )  there are 3 folders 
1 - original image (source image).
2- face_text file ( boundaries of face mask ).
3-test label Image (image for pose shape).

the question is where is the target person Image to wrap it with new pose ?

**Comment by stanleyshly** [comment]
There is no target image, the background is trained into the model. In order to wrap to a new pose, you simplify feed your target post the same way training data is prepared, and you run the model, and the output will be new images with retargeted poses.

**Comment by justinjohn0306** [comment]
and what would be the data/file structure/tree?
for source and target dataset folder?
